src/app/modules/generator/components/action/action.component.scss

```
@@ -0,0 +1,10 @@
+:host {
+    margin-bottom: 32px;
+
+    display: flex;
+    justify-content: space-between;
+
+    app-button + app-button{
```

why not a generic child range like 
```suggestion
    app-button:nth-child(1):nth-child(2){
```
or a negative child range
```suggestion
    app-button:nth-child(-n+2){
```

as suggested in [nthmaster](http://nthmaster.com/)